src/qt/bitcoin.cpp

```diff
@@ -357,9 +357,9 @@ void BitcoinApplication::createSplashScreen(const NetworkStyle *networkStyle)
 {
     SplashScreen *splash = new SplashScreen(m_node, 0, networkStyle);
     // We don't hold a direct pointer to the splash screen after creation, but the splash
-    // screen will take care of deleting itself when slotFinish happens.
+    // screen will take care of deleting itself when finish() happens.
     splash->show();
-    connect(this, &BitcoinApplication::splashFinished, splash, &SplashScreen::slotFinish);
+    connect(this, &BitcoinApplication::splashFinished, splash, &SplashScreen::finish);
```

Comment:
Could just
```
connect(this, &BitcoinApplication::splashFinished, splash, &SplashScreen::deleteLater);
```
or is there a good reason to call `hide()`?

Reply:
When #8231 was merged there was no `deleteLater()` call.

Reply:
Tested on macOS 10.13.6: if the splash screen is minimized, neither `deleteLater()` nor `hide(); deleteLater()` can remove it from the Dock.
It seems the simplest way is to not touch the patch from #8231.

Reply:
@promag Moreover, IIUC, 34ebceb25a40a2160349656265951cbe9a5b0f34 commit has been effectively disabled by e4f126a7ba66e7317718c30276dff6db92dc1986.
What do you think about this?

Reply:
Just tested my suggestion above with Qt 5.11 on macOS 10.14.1 and the following are good:
 - closing the app while splash is open
 - splash goes away from the dock even if it's minimized.

What's your Qt version?

Reply:
@promag 
> What's your Qt version?

Qt 5.11.2

Reply:
@hebasto with my above suggestion I can't reproduce #7718 in my system. Maybe this is was macOS issue?

Reply:
@promag 
>  Maybe this is was macOS issue?

Agree. As there are systems subject to #7718 it is better to not touch the patch from #8231, IMO.